sonic_package_manager/manager.py

```diff
@@ -930,14 +960,14 @@ def _install_cli_plugin(self, package: Package, command: str):
         image_plugin_path = package.manifest['cli'][command]
         if not image_plugin_path:
             return
-        host_plugin_path = self._get_cli_plugin_path(package, command)
```

Comment:
Will you be removing _get_cli_plugin_path method ?

Reply:
Right, thanks